app/models/agents/telegram_agent.rb

```diff
@@ -13,7 +13,7 @@ class TelegramAgent < Agent
 
       It is assumed that events have either a `text`, `photo`, `audio`, `document` or `video` key. You can use the EventFormattingAgent if your event does not provide these keys.
 
-      The value of `text` key is sent as a plain text message.
+      The value of `text` key is sent as a plain text message. Also you can specify `parse_mode`, how Telegram will parse the message, - the value should be `html` or `markdown`.
```

Comment:
Maybe

"You can also tell Telegram how to parse the message with `parse_mode`, set to either `html` or `markdown`."